static/js/taxpayer_creation_form_logic.js

```diff
@@ -51,19 +67,17 @@ function country_change(){
     else if ($("#id_address_form-country").val() == "US"){
         $(".usa").show();
         $(".arg").hide();
-        console.log("no es required");
         arg_unrequired();
         empty_arg();
     }
     else{
-        console.log("nothing to do")
+        // console.log("nothing to do")
```

Comment:
Maybe you are putting this console logs in a way to "debug" but it is not totally correct to display this in production. Also if this else block just print a console log maybe you should remove it.